src/content/chainlink-automation/tutorials/time-based-upkeep.mdx

```diff
@@ -0,0 +1,211 @@
+
+<TabsContent client:visible>
+<Fragment slot="tab.1">Verify your contract</Fragment>
+<Fragment slot="tab.2">Add the ABI manually</Fragment>
+<Fragment slot="panel.1">
+
+Verify your contract directly in the blockchain explorer by adding your contract's source code and other required information in the blockchain explorer UI. For example:
+
+- Verify contracts deployed to Sepolia on the [testnet version of Etherscan](https://sepolia.etherscan.io/verifyContract).
+- Verify contracts deployed to Polygon Mumbai testnet on the [testnet version of Polygonscan](https://mumbai.polygonscan.com/verifyContract).
+
+After you successfully verify your smart contract, the Chainlink Automation App fetches its ABI automatically when you [register an upkeep](#register-a-new-upkeep):
+
+<ClickToZoom src="/images/automation/qs-time-based-upkeep/abi-fetched.png" alt="ABI fetched automatically" />
+
+</Fragment>
+<Fragment slot="panel.2">
+
+If you do not want to verify the contract on chain, paste the contract's ABI into the corresponding text box in the Chainlink Automation App while you are registering the upkeep.
+
+1.  In Remix, within the Solidity Compiler tab, copy the ABI here:
+
+    <ClickToZoom src="/images/automation/qs-time-based-upkeep/get-abi-remix.png" alt="Remix Copy ABI" />
+
+1.  In the Chainlink Automation App, paste the ABI into the ABI field:
+
+    <ClickToZoom
+      src="/images/automation/qs-time-based-upkeep/couldnt-fetch-abi-ui.png"
+      alt="Chainlink Automation App ABI field"
+    />
+
+</Fragment>
+</TabsContent>
+
+## Register a new Upkeep
+
+<div class="remix-callout">
+  <a href="https://automation.chain.link">Open the Chainlink Automation App</a>
+</div>
+
+1. **Click the Register New Upkeep button**
+
+   <ClickToZoom src="/images/contract-devs/automation/auto-ui-home.png" alt="Click Register New Upkeep" />
+
+1. **Connect your wallet** using the _Connect Wallet_ choose a network. For a list of supported networks, see the [Supported Blockchain Networks](/chainlink-automation/supported-networks) section. The Chainlink Automation App also lists the currently supported networks.
+
+   <ClickToZoom src="/images/contract-devs/automation/auto-ui-wallet.png" alt="Connect With Metamask" />
+
+1. Select the time-based trigger.
+
+1. Provide the address of your deployed contract. If your contract is unverified, paste the ABI into the corresponding text box.
+
+1. If you're using the `Counter` contract, add a value for the function input in the Chainlink Automation App:
+
+   <ClickToZoom src="/images/automation/qs-time-based-upkeep/update-amount.png" alt="Function input" />
+
+   Each time Automation triggers your function, based on the schedule you give it, it will run the `updateCounter` function, which increments the `counter` variable using the value you enter for `updateAmount`.
```

Comment:
```suggestion
   Each time Automation triggers your function, based on the schedule you give it, it will run the `updateCounter` function which increments the `counter` variable using the value you enter for `updateAmount`.
```